test/sagemaker_tests/pytorch/inference/conftest.py

```diff
@@ -250,18 +250,23 @@ def fixture_ecr_image(docker_registry, docker_base_name, tag):
 def skip_by_device_type(request, use_gpu, instance_type, accelerator_type):
     is_gpu = use_gpu or instance_type[3] in ['g', 'p']
     is_eia = accelerator_type is not None
+    is_neuron = instance_type == 'ml.inf1.xlarge'
```

Comment:
The instance type won't always be `inf1.xlarge`, right?

Could this instead be `instance_type.startswith("ml.inf")`?

Reply:
assign_sagemaker_remote_job_instance_type is assigning only ml.inf1.xlarge for neuron but yes will change as we have the option to provide --instance-type also.